pages/signup.vue

```diff
@@ -0,0 +1,245 @@
+<template>
+    <v-container no-gutters fluid fill-height class="pa-0 ma-0">
+      <v-row class="d-flex flex-column align-center justify-center">
+        <v-col class="pa-0 ma-0">
+          <v-sheet
+            class="section d-flex align-center"
+            style="padding-bottom: 200px"
+          >
+            <!-- Right side for login component -->
+            <v-col
+              cols="12"
+              xs="12"
+              sm="10"
+              md="10"
+              lg="4"
+              xl="3"
+              xxl="3"
+              class="d-flex flex-column justify-start align-start"
+            >
+                <h1 class="login-text mx-auto">Sign Up</h1>
+                <div class="text-input-wrapper">
+                    <input class="duolingo-text-input" v-model="email" data-test="email-input" type="text" placeholder="Email or username">
```

Comment:
I'm going to merge my styling fixes for the login page: https://github.com/CapSnCrunch/learn-anything/pull/2

You can go ahead and merge master into your branch and copy those instead so we don't have to go back and fix all this a second time.